pyogrio/tests/test_raw_io.py

```diff
@@ -155,19 +155,20 @@ def test_read_no_geometry_no_columns_no_fids(naturalearth_lowres):
         )
 
 
-def test_read_columns(naturalearth_lowres):
-    columns = ["NAME", "NAME_LONG"]
-    meta, _, geometry, fields = read(
-        naturalearth_lowres, columns=columns, read_geometry=False
-    )
-    array_equal(meta["fields"], columns)
-
-    # Repeats should be dropped
-    columns = ["NAME", "NAME_LONG", "NAME"]
-    meta, _, geometry, fields = read(
+@pytest.mark.parametrize(
+    "descr, columns, exp_columns",
+    [
+        ("all", None, ["pop_est", "continent", "name", "iso_a3", "gdp_md_est"]),
+        ("case_sensitive", ["NAME", "NAME_LONG"], []),
```

Comment:
```suggestion
        ("case_sensitive", ["NAME"], []),
```

("name_long" is not in the columns anyway, so just a bit confusing to include here. This is a leftover from long ago when the raw naturalearth data was used which has capital names and did have a NAME_LONG column .. which wasn't catched because of the missing assert)